### PR TITLE
chore: update wordings in bot template

### PR DESCRIPTION
**Pull request by blackchoey**
Based on Tim's feedback to our sample project

### CODE REVIEW

File: templates/scenarios/notification-restify-ts/bot/src/index.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { CardData } from "./cardModels";
 // Create HTTP server.
 const server = restify.createServer();
 server.listen(process.env.port || process.env.PORT || 3978, () => {
-  console.log(`\nBot Started, ${server.name} listening to ${server.url}`);
+  console.log(`\nApp Started, ${server.name} listening to ${server.url}`);
```
Comment (swatDong) [inline templates/scenarios/notification-restify-ts/bot/src/index.ts:10]: Any change to `import { bot } from "./internal/initialize";` ?

File: templates/scenarios/command-and-response-ts/bot/README.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ import { HelloWorldCommandHandler } from "../helloworldCommandHandler";
 import { DoSomethingCommandHandler } from "../doSomethingCommandHandler";
 import { ConversationBot } from "@microsoft/teamsfx";
 
-const commandBot = new ConversationBot({
```
Comment (dooriya) [inline templates/scenarios/command-and-response-ts/bot/README.md:156]: In notification samples, we use the following code:
```
export const bot = new ConversationBot({
...
});
```
Shall we unify the naming for all `ConversationBot` in our samples?

Comment (blackchoey) [reply templates/scenarios/command-and-response-ts/bot/README.md:156]: Thanks, will change the variable name to notificationApp in notification templates. There's also feedback about renaming ConversationBot, we can track the SDK changes separately.

File: templates/scenarios/notification-restify-ts/bot/src/index.ts (hunk 1 above)
Comment (blackchoey) [reply templates/scenarios/notification-restify-ts/bot/src/index.ts:10]: Thanks, updated

**Review by dooriya — APPROVED** [review]

**Review by MuyangAmigo — APPROVED** [review]

**Comment by blackchoey** [comment]
Test passed for notification js template. Will test other templates when we transform them into a valid V3 project.

**Pull request merged**